Dockerfile

```diff
@@ -158,44 +178,46 @@ RUN rm /etc/apache2/sites-enabled/000-default.conf
 COPY --from=builder /tmp/local/ /opt/perl/local/
 ENV PERL5LIB="/opt/product-opener/lib/:/opt/perl/local/lib/perl5/"
 ENV PATH="/opt/perl/local/bin:${PATH}"
+# Create writable dirs and change ownership to www-data
+RUN \
+    mkdir -p var/run/apache2/ && \
+    chown www-data:www-data var/run/apache2/ && \
+    for path in users products product_images orgs new_images logs; do \
+        mkdir -p /mnt/podata/${path}; \
+        chown www-data:www-data /mnt/podata/${path}; \
+    done && \
+    # Create symlinks of data files in /mnt/podata (because we currently mix data and conf data)
+    for path in ecoscore emb_codes forest-footprint ingredients lang packager-codes po taxonomies templates; do \
+        ln -sfT /opt/product-opener/${path} /mnt/podata/${path}; \
+    done
```

Comment:
I'm convinced this still fits better in the init script. What would be best is to not need those in the first place, i.e get rid of the /mnt/podata symlinking, and not need to change the permissions. How we are going to achieve this, I'm not sure yet.

Reply:
Sounds good